Dockerfile

```
@@ -1,4 +1,4 @@
-FROM php:7.1-cli AS dev
+FROM php:7.4-cli AS dev
```

Is this change intended? The required version of php in `composer.json` is `>=7.1`.

```suggestion
FROM php:7.1-cli AS dev
```